components/map/EventMarker.js

```diff
@@ -0,0 +1,54 @@
+import L from "leaflet";
+import { Marker, Popup } from "react-leaflet";
+import { ReactMarkdown } from "react-markdown/lib/react-markdown";
+import Link from "@components/Link";
+
+export default function EventMarker({event}) {
+  // Custom component for rendering links within ReactMarkdown
+  const LinkRenderer = ({ href, children }) => (
+    <Link href={href}>
+      {children}
+    </Link>
+  );
+
+  return (
+    <Marker
+      icon={L.icon({
+        className: "rounded-full",
+        iconUrl: `/placard.png`,
```

Comment:
This can use standard string quotes

```suggestion
        iconUrl: "/placard.png",
```